recipe/meta.yaml

```
@@ -164,6 +126,23 @@ requirements:
     - alsa-lib                           # [linux]
     - krb5
     - harfbuzz                           # [linux]
+    # Adding the following xcb-util run requirements resolves issue
+    # that may exhibit as segfault due to missing libraries
+    # in some lighter weight linux installations
+    # https://github.com/conda-forge/qt-main-feedstock/issues/32
```

```suggestion
    # The xcb-util packages are necessary to build the Qt xcb plugin.
```

I think a comment like this is a better suited for the `host` section (you can reedit it if you want).

---

done in the next push.